chevah/compat
# Refactor isUserInGroup to getGroupForUser to also get the name of the matched group.

Scope
=====

This is just a refactor of the existing isUserInGroup functionality so also return the name of the first matched groupl

It should make it easier to identify the name of the group without any extra system calls.

Changes
=======

Use the same implementation but rename the API to getGroupForUser and return the name of the matched group.

In theory, the API could also return the group ID or UID on windows...but I don't see any use of that, so I will let that change for a later time.

As a drive by I have updated the build briink.sh script.


How to try and test the changes
===============================

reviewers: @danuker 

Let me know if the changes make sense.

The API is used here https://github.com/chevah/server/pull/1772

-----

This PR was created without a Trac ticket as this is a public repo and I think that for public repo we should do as much work as possible in the public.
